xml/appendix_values_yaml.xml

```diff
@@ -0,0 +1,1791 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE appendix
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+
+<appendix xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="app.kvm">
+ <title>Appendix</title>
+ <info>
+      <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+        <dm:bugtracker>
+          </dm:bugtracker>
+      </dm:docmanager>
+    </info>
+    
+    <para>
+        This appendix contains copies of the complete 
+        <filename>values.yaml</filename> files for the SCF and UAA deployments.
+    </para>
+    
+ <sect1 xml:id="app.scf.values.yaml">
+  <title>Complete &scf; values.yaml file</title> 
+
+<para>
+    This is the &scf; <filename>values.yaml</filename> configuration
+    file that is shipped with the &helm; charts. It provides a complete reference
+    of configuration options for &scf;, and their default settings.
```

Comment:
Maybe _an up-to-date version can be obtained via `helm inspect values suse/cf`_ or something along those lines?

Reply:
Yes, good idea, the explanation needs more explaining.